src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Models/MetaDataFileReader.cs

```diff
@@ -87,15 +88,15 @@ public MetaRow GetMetaRow(IReadOnlyList<string> rowValues)
         });
 
         var indicatorGroups = indicatorRows
-            .GroupBy(row => row.IndicatorGrouping)
+            .GroupBy(row => row.IndicatorGrouping!)
             .ToDictionary(
                 rows => rows.Key,
                 rows => new IndicatorGroup(rows.Key, subject.Id));
 
         return indicatorRows
             .Select(row =>
             {
-                var indicatorGroup = indicatorGroups.GetValueOrDefault(row.IndicatorGrouping)!;
+                var indicatorGroup = indicatorGroups.GetValueOrDefault(row.IndicatorGrouping!)!;
```

Comment:
I guess that this could probably be `indicatorGroups[row.IndicatorGrouping]` if we're so sure that it's going to be not null.